app/helpers/curr_weather_helper.rb

```diff
@@ -0,0 +1,24 @@
+module CurrWeatherHelper
+  # from here on get all data out of json(curr_weather_data)
+  def get_curr_weather_data(curr_weather_data)
+    weather_pic(curr_weather_data)
+    @temp = ((curr_weather_data['main']['temp']).to_f - 273.15).round(1)
```

Comment:
When you're converting the temperate, it should be its own method with a descriptive name, so we know what you're trying to do. Otherwise, it's hard to tell what's happening here.